test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -201,6 +201,17 @@ Describe "Basic FileSystem Provider Tests" -Tags "CI" {
             { Move-Item -Path $src -Destination $dest -ErrorAction Stop } | Should -Throw -ErrorId 'MoveItemArgumentError,Microsoft.PowerShell.Commands.MoveItemCommand'
         }
 
+        It 'Verify Move-Item fails for destination to be same as source without directory separator' {
+            try {
+                Push-Location $TestDrive
+                New-Item -ItemType Directory -Path 'Empty'
+                { Move-Item -Path ./Empty/ -ErrorAction Stop } | Should -Throw -ErrorId 'MoveItemArgumentError,Microsoft.PowerShell.Commands.MoveItemCommand'
```

Comment:
Is there already a test for  `./Empty` (to avoid regressions)?

Could the test name be more specific like `Verify Move-Item don't allow moving a directory into itself or its sub-directory`?